src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,21 +2,24 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.debug("Method \'login\' was called. Params: login={}", login);
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
-            throw new AuthenticationException("Username or password are incorrect");
+            logger.error("Username or password are incorrect");
         }
```

Comment:
The `login` method should throw an `AuthenticationException` when the password is incorrect, as indicated by the method signature. Consider adding a throw statement here.